### PR TITLE
NOJIRA add helper scripts for working with aws glue jobs

### PR DESCRIPTION
**Pull request by carwynmoj**
Summary of changes

Added the following scripts
 * push-jar - builds and pushes the project jar to an s3 folder uri
 * tail-glue-logs - tail the logs for a given job name
 * start-and-tail-job - starts a given job name and then tails the logs

### CODE REVIEW

File: scripts/push-jar [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+echo
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+./gradlew clean shadowJar
+
+echo
+echo "Uploading jar to $s3_uri"
+echo
+aws s3 cp ./build/libs/digital-prison-reporting-jobs-1.0-SNAPSHOT-all.jar $s3_uri/digital-prison-reporting-jobs-1.0-SNAPSHOT-all.jar
```
Comment (kartthik-n) [inline scripts/push-jar:35]: can you please make the version 1.0-SNAPSHOT as variable? this will change as per the release

Comment (carwynmoj) [reply scripts/push-jar:35]: Ahh yeah, good spot. Afaik we're not doing this dynamically yet but I'll revise this to match on the static parts of the jar name instead.

Comment (carwynmoj) [reply scripts/push-jar]: Pushed up changes now which also includes a fix to ensure the gradle build is run in the correct directory.

File: scripts/push-jar [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+echo "Building jar"
+echo
+
+cd $script_dir/../
+./gradlew clean shadowJar
+
+echo
+echo "Uploading jar to $s3_uri"
+echo
+aws s3 cp ./build/libs/digital-prison-reporting-jobs-*-all.jar $s3_uri/digital-prison-reporting-jobs-1.0-SNAPSHOT-all.jar
```
Comment (kartthik-n) [inline scripts/push-jar:36]: do we need to specify the destination file name here? can we simplify like this?

aws s3 cp ./build/libs/digital-prison-reporting-jobs-*-all.jar $s3_uri/

Comment (carwynmoj) [reply scripts/push-jar:36]: You're quite right. This is a hangover from a version I knocked up for myself a while back. Should've read the whole line.

Comment (carwynmoj) [reply scripts/push-jar]: Fix pushed for this (dest filename has been removed) so resolving this.

**Review by kartthik-n — APPROVED** [review]

**Pull request merged**